src/php/php_server.rs

```diff
@@ -57,22 +61,41 @@ pub(crate) fn start() -> PhpServer {
         )),
     }
 
+    let process_pid = process.id();
+
     ctrlc::set_handler(move || {
         info!("Stopping PHP process... ");
 
         #[cfg(not(target_os = "windows"))]
-        {
-            let pid = process.id();
-            stop_process::stop(pid.to_string().as_ref()); // Stop fpm children
-        }
+            {
+                let pid = process.id();
+                stop_process::stop(pid.to_string().as_ref()); // Stop fpm children
+            }
```

Comment:
There's no need to indent here, is there?

```suggestion
        {
            let pid = process.id();
            stop_process::stop(pid.to_string().as_ref()); // Stop fpm children
        }
```